crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -472,6 +477,31 @@ impl RpcServerConfig {
         self
     }
 
+    // Confiugures the http server with cors domain
+    pub fn with_http_cors_domain(mut self, config: ServerBuilder, domains: Vec<&str>) -> Self {
```

Comment:
I would rather have this as a separate field (cors_domans: Option<...>)and then this is created in `build`

Reply:
Could you please elaborate on this? I am a bit confused